ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -623,6 +623,612 @@ public final class AliasTokens {
         }
     }
 
+    // MARK: Shared Colors
+
+    public enum SharedColorsTokens: CaseIterable {
```

Comment:
The latest designs in Figma have moved this out of alias tokens and into the Avatar itself. Feel free to ping me offline if you have questions or need any pointers!